src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
Betamax won't work with https right now, or maybe some changes to client should be done.
Actually, I've found that Betamax should work well on https with apache httpclient, but it's over 700kb without its own dependencies and I think you don't need it.)
Maybe with betamax update the situation will change.